metadata/src/test/java/org/apache/kafka/controller/QuorumControllerTest.java

```diff
@@ -1318,7 +1318,52 @@ public void testFatalMetadataReplayErrorOnActive() throws Throwable {
             assertEquals(NullPointerException.class,
                     controlEnv.fatalFaultHandler().firstException().getCause().getClass());
             controlEnv.fatalFaultHandler().setIgnore(true);
-            controlEnv.metadataFaultHandler().setIgnore(true);
+        }
+    }
+
+    private final static List<ApiMessageAndVersion> CORRUPT_RECORD =
```

Comment:
nit: perhaps better to call this `INVALID_RECORD` with a brief comment mentioning the topicId is not set. Also, since it's not used in any other tests, maybe we can keep it local to `testFatalMetadataReplayErrorOnStandbys`? It's always possible to factor it out if it does get used in other tests.